### PR TITLE
moving changes for KPI options to featured section and minor fixes

### PR DESCRIPTION
**Pull request by ChandlerDeKock**

### CODE REVIEW

File: docs/developers/devmining-reqs.md [hunk 1]
```diff
@@ -32,7 +32,7 @@ To perform this process, each synthetic token must have an available price feed
 
 ### Send deployment information to the UMA core team
 
-The final step required to be added to developer mining rewards is to alert a member of the UMA core team of your successful deployment. To do this, fill out the google form [here](https://docs.google.com/forms/d/e/1FAIpQLSfhrCopRz7nUSbBQZB75j8yGqnXbOzRW68Oe6-uFuNVSvqrOQ/viewform).
```
Comment (smb2796) [inline docs/developers/devmining-reqs.md:35]: I don't think this should be removed. We still use this form

File: src/css/custom.css [hunk 2]
```diff
@@ -57,6 +57,17 @@ html[data-theme=dark] .header-github-link:before {
   background: url("data:image/svg+xml;charset=utf-8,%3Csvg viewBox='0 0 24 24' xmlns='http://www.w3.org/2000/svg'%3E%3Cpath fill='%23fff' d='M12 .297c-6.63 0-12 5.373-12 12 0 5.303 3.438 9.8 8.205 11.385.6.113.82-.258.82-.577 0-.285-.01-1.04-.015-2.04-3.338.724-4.042-1.61-4.042-1.61C4.422 18.07 3.633 17.7 3.633 17.7c-1.087-.744.084-.729.084-.729 1.205.084 1.838 1.236 1.838 1.236 1.07 1.835 2.809 1.305 3.495.998.108-.776.417-1.305.76-1.605-2.665-.3-5.466-1.332-5.466-5.93 0-1.31.465-2.38 1.235-3.22-.135-.303-.54-1.523.105-3.176 0 0 1.005-.322 3.3 1.23.96-.267 1.98-.399 3-.405 1.02.006 2.04.138 3 .405 2.28-1.552 3.285-1.23 3.285-1.23.645 1.653.24 2.873.12 3.176.765.84 1.23 1.91 1.23 3.22 0 4.61-2.805 5.625-5.475 5.92.42.36.81 1.096.81 2.22 0 1.606-.015 2.896-.015 3.286 0 .315.21.69.825.57C20.565 22.092 24 17.592 24 12.297c0-6.627-5.373-12-12-12'/%3E%3C/svg%3E") no-repeat;
 }
 
+.header-web-link:before {
```
Comment (smb2796) [inline src/css/custom.css:60]: I'm not sure I love the choice of using the prototypical WWW logo for the UMA home page. Could we use one of the UMA logos from [here](https://github.com/UMAprotocol/website/tree/master/docs/press-kit/logos)?

Also the sizing/spacing looks a little wonky.

**Review by smb2796 — CHANGES_REQUESTED** [review]
Thanks for putting this up @ChandlerDeKock! Left a couple of comments

**Comment by smb2796** [comment]
The logo doesn't show up for me (but is clickable) 
image

**Comment by ChandlerDeKock** [comment]
The issue was the directory the build was looking in. Should be fixed now

**Review by smb2796 — APPROVED** [review]
Looks great!

**Pull request merged**